cv32e40p/sim/uvmt/Makefile

```diff
@@ -52,5 +52,18 @@ DPI_DASM_SPIKE_REPO  ?= https://github.com/riscv/riscv-isa-sim.git
 CV_SW_TOOLCHAIN      ?= /opt/riscv
 CV_SW_PREFIX         ?= " "
 
+YOUR_GCC_VERSION     := $(shell $(CV_SW_TOOLCHAIN)/bin/$(CV_SW_PREFIX)gcc -dumpversion)
+REQD_GCC_VERSION     := 13.0.0
```

Comment:
Our gcc version is `riscv32-corev-elf-gcc ('corev-openhw-gcc-centos7-20221021') 12.0.1 20220422 (experimental)`, not `13.0.0`.

Reply:
Yes, but if you issue the command `gcc -dumpversion` you get something like 12.0.1, not riscv32-corev-elf-gcc ('corev-openhw-gcc-centos7-20221021') 12.0.1 20220422 (experimental).   AFAIK, the important information, at least as far as our usage is concerned is whatever `dumpversion` prints.  Let me know if you think that is not correct.

Reply:
Yes, but why is `13` required when `12` works?

Reply:
Yes >= 12.* should be good enough in the comparison.